invenio_communities/communities/services/service.py

```diff
@@ -463,18 +448,27 @@ def featured_delete(
     # Deletion workflows
     #
     @unit_of_work()
-    def delete_community(self, identity, id_, data, expand=False, uow=None):
+    def delete_community(
+        self, identity, id_, data=None, revision_id=None, expand=False, uow=None
```

Comment:
the `revision_id` is accepted **but ignored** because the old signature ([and the resource actually provides a value from the `if_match` request context header](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/resources/records/resource.py#L144))

not sure if that's the way to go?

Reply:
We should [check the revision id as it is done in the RecordService](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/records/service.py#L491)